airport/airport-pojo-bl/src/main/java/berlin/tablecodes/assets/AssetType.java

```diff
@@ -44,6 +46,22 @@
     @CompositeKeyMember(1)
     private String name;
     
+    @IsProperty
+    @MapTo
+    @Title(value = "Condition Rating", desc = "Condition rating for an asset")
```

Comment:
Condition rating for assets of this type.

Reply:
fixing this